content/hi/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,19 @@
+---
+title: काम का बोझ
+id: workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
+short_description: >
+   काम का बोझ एक ऐप्लिकेशन है जो Kubernetes पर चल रहा है।
+
+aka: 
+tags:
+- मौलिक
+---
+   काम का बोझ एक ऐप्लिकेशन है जो Kubernetes पर चल रहा है।
+
+<!--more--> 
+
+विभिन्न मूल वस्तुएँ जो विभिन्न प्रकार के या भागों को प्रतिनिधित करती हैं, उनमें DaemonSet, Deployment, Job, ReplicaSet और StatefulSet वस्तुएँ शामिल हैं।
```

Comment:
You'll need to transliterate objects i.e. write objects in devnagari script, not translate it. 
```suggestion
विभिन्न मूल ऑब्जेक्ट जैसे DaemonSet, Deployment, Job, ReplicaSet और StatefulSet कार्यभार के विभिन्न प्रकार या भागों को प्रतिनिधित करती हैं।
```